### PR TITLE
DHFPROD-7741: Component Replacement: Treeselect

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by Sanjeevani19 — COMMENTED** [review]
@iveyMarklogic Please make sure that the PR title is in correct format.

**Comment by iveyMarklogic** [comment]
@Sanjeevani19 I think the PR has the correct title format

File: marklogic-data-hub-central/ui/e2e/cypress/support/components/matching/ruleset-single-modal.tsx [hunk 1]
```diff
@@ -1,19 +1,17 @@
 class RulesetSingleModal {
 
   selectPropertyToMatch(property: string) {
-    cy.get("[class^=\"entity-property-tree-select_matchTypeSelect\"] > .ant-select-selection").trigger("mouseover").click();
+    cy.get(".rc-tree-select-selector").trigger("mouseover").click();
```
Comment (csilva-ml) [inline marklogic-data-hub-central/ui/e2e/cypress/support/components/matching/ruleset-single-modal.tsx:4]: This is a really great selector improvement!

**Review by yubiselmarklogic — DISMISSED** [review]

**Review by mrodrigu-marklogic — CHANGES_REQUESTED** [review]
There is a slight difference on the container that puts the horizontal scroll bar when you don't have children there, aside from that LGTM 

image

**Comment by iveyMarklogic** [comment]
@mrodrigu-marklogic can you check again

**Review by yubiselmarklogic — APPROVED** [review]

**Review by mrodrigu-marklogic — APPROVED** [review]

**Pull request merged**